tests/factory/tools_test.py

```diff
@@ -0,0 +1,84 @@
+"""Unit test for factory tools."""
+import pytest
+import ipaddress
+
+from ostorlab.agent.message import message
+
+from agent.tsunami.factory import preapre_tagets_tools as tools
+
+
+msg_1 = message.Message.from_data(
+    selector="v3.asset.domain_name", data={"name": "ostorlab.co"}
+)
+
+msg_2 = message.Message.from_data(
+    selector="v3.asset.ip.v4", data={"version": 4, "host": "0.0.0.0"}
+)
+
+msg_3 = message.Message.from_data(
+    selector="v3.asset.domain_name.service",
+    data={"name": "ostorlab.co", "port": 6000, "schema": "https"},
+)
+
+msg_4 = message.Message.from_data(
```

Comment:
Please inline these message for better readability.

Reply:
I applied the black style